pypeit/spectrographs/keck_lris.py

```diff
@@ -179,13 +179,14 @@ def compound_meta(self, headarr, meta_key):
             return binning
         elif 'lampstat' in meta_key:
             idx = int(meta_key[-2:])
-            curr_date = time.Time(self.get_meta_value(headarr, 'mjd'), format='mjd')
-            #except:
-            #    msgs.warn('No mjd in header. You either have bad headers, '
-            #              'or incorrectly specified the wrong spectrograph, '
-            #              'or are reading in other files from your directory.  '
-            #              'Using 2022-01-01 as the date for parsing lamp info from headers')
-            #    curr_date =  time.Time("2022-01-01", format='isot')
+            try:
```

Comment:
I worry that a `run_pypeit` run will proceed with
incorrect MJD and yield bad heliocentric corrections, etc.

So, we want this to trip for `run_pypeit`

Reply:
Okay, well I think the solution here is to assign the MJD to a value like -1 that will cause PypeIt to crash but not setup to crash, since our mantra is that setup should never crash.